### PR TITLE
BZ 1583143 - Add info on configuring zone labels

### PR DESCRIPTION
**Pull request by ghost**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1583143

### CODE REVIEW

File: install_config/configuring_openstack.adoc [hunk 1]
```diff
@@ -12,21 +12,15 @@ toc::[]
...
 
-[cols='1,3']
-|===
-
-| member
-| For creating assets(instances, networking ports, floating ips, volumes, and so
-on.) you need the member role for the tenant.
-
-|===
+Member:: For creating assets(instances, networking ports, floating IPs, volumes, and so
```
Comment (kalexand-rh) [inline install_config/configuring_openstack.adoc]: Rework to remove the parentheses.

File: install_config/configuring_openstack.adoc [hunk 2]
```diff
@@ -166,12 +154,11 @@ kubeletArguments:
   cloud-config:
     - "/etc/cloud.conf"
 ----
-<1> Name of the OpenStack instance where the node runs (i.e., name of the virtual machine)
-====
+<1> Name of the OpenStack instance where the node runs (that is, the name of the virtual machine)
```
Comment (kalexand-rh) [inline install_config/configuring_openstack.adoc]: Maybe:
<1> The RFC1123-compliant OpenStack instance name of the node host.

And then a note that says:
If the `nodeName` does not match the OpenStack instance name, the cloud provider integration will not work.

File: install_config/configuring_openstack.adoc [hunk 3]
```diff
@@ -184,3 +171,12 @@ container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
 [[openstack-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[openstack-configuring-zone-labels]]
+=== Configuring Zone Labels for Dynamically Created OpenStack PVs 
+
+Administrators now have the option to configure zone labels for dynamically
```
Comment (kalexand-rh) [inline install_config/configuring_openstack.adoc]: s/Administrators now have the option to configure/Administrators can

File: install_config/configuring_openstack.adoc [hunk 4]
```diff
@@ -184,3 +171,12 @@ container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
...
+created OpenStack PVs. This option is useful if the OpenStack Cinder zone name
+does not match the compute zone names. For example, if there is only one Cinder
```
Comment (kalexand-rh) [inline install_config/configuring_openstack.adoc]: s/names. For/names, for

File: install_config/configuring_openstack.adoc [hunk 5]
```diff
@@ -184,3 +171,12 @@ container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
...
+zone and many compute zones. Administrators can create Cinder volumes
+dynamically and then check the labels.
```
Comment (kalexand-rh) [inline install_config/configuring_openstack.adoc]: I think that you do need to explain how to add the label. I hope you get the info you asked for in the BZ.  :)

**Comment by qinpingli** [comment]
LGTM

**Pull request merged**